src/sagas/communities/createCommunity/createCommunity.saga.test.ts

```diff
@@ -53,11 +42,19 @@ describe('createCommunitySaga', () => {
         SocketActionTypes.CREATE_COMMUNITY,
```

Comment:
I don't fully get why this provide is working? SocketActionTypes.CREATE_COMMUNITY is an enum value (not a function)

Reply:
I think we always test it like this?

Reply:
Ok, part of the code is hidden by GitHub. Nevermind...